config/schema/elasticsearch_types/life_saving_maritime_appliance_service_station.json

```diff
@@ -0,0 +1,297 @@
+        "label": "Callegari",
+        "value": "callegari"
+      },
+      {
+        "label": "Crewsaver",
+        "value": "crewsaver"
+      },
+      {
+        "label": "CSM Liferafts (Shanghai Star)",
+        "value": "csm-liferafts-shanghai-star"
+      },
+      {
+        "label": "Delta Lifejackets",
+        "value": "delta-lifejackets"
+      },
+      {
+        "label": "Delta RIBs",
+        "value": "delta-ribs"
+      },
+      {
+        "label": "DSB",
+        "value": "dsb"
+      },
+      {
+        "label": "Duarry",
+        "value": "duarry"
+      },
+      {
+        "label": "Eurovinil",
+        "value": "eurovinil"
+      },
+      {
+        "label": "Fujikura MES",
+        "value": "fujikura-mes"
+      },
+      {
+        "label": "Halmatic RIBs",
+        "value": "halmatic-ribs"
+      },
+      {
+        "label": "Hansen Protection - UK",
+        "value": "hansen-protection-uk"
+      },
+      {
+        "label": "Harding Watercraft RIBs",
+        "value": "harding-watercraft-ribs"
+      },
+      {
+        "label": "International Safety Group",
+        "value": "international-safety-group"
+      },
+      {
+        "label": "ISP Challenger",
+        "value": "isp-challenger"
+      },
+      {
+        "label": "Jiangsu Haining",
+        "value": "jiangsu-haining"
+      },
+      {
+        "label": "K2 Lifejackets",
+        "value": "k2-lifejackets"
+      },
+      {
+        "label": "KI-Elements Lifejackets",
+        "value": "ki-elements-lifejackets"
+      },
+      {
+        "label": "Lalizas",
+        "value": "lalizas"
+      },
+      {
+        "label": "Lifeguard",
+        "value": "lifeguard"
+      },
+      {
+        "label": "Liferaft Systems Australia",
+        "value": "liferaft-systems-australia"
+      },
+      {
+        "label": "Marin Ark 1",
+        "value": "marin-ark-1"
+      },
+      {
+        "label": "Marin Ark 2",
+        "value": "marin-ark-2"
+      },
+      {
+        "label": "Maritime Partner Craft",
+        "value": "maritime-partner-craft"
+      },
+      {
+        "label": "Matchau Marine",
+        "value": "matchau-marine"
+      },
+      {
+        "label": "Mulder and Rijke",
+        "value": "mulder-and-rijke"
+      },
+      {
+        "label": "Mullion",
+        "value": "mullion"
+      },
+      {
+        "label": "Narwhal",
+        "value": "narwhal"
+      },
+      {
+        "label": "Ningbo Zhenhua",
+        "value": "ningbo-zhenhua"
+      },
+      {
+        "label": "Ningbo Haifeng",
+        "value": "ningbo-haifeng"
+      },
+      {
+        "label": "Ningbo Neptune",
+        "value": "ningbo-neptune"
+      },
+      {
+        "label": "Norsafe Boats",
+        "value": "norsafe-boats"
+      },
+      {
+        "label": "Ocean Safety (own brand)",
+        "value": "ocean-safety-own-brand"
+      },
+      {
+        "label": "Ocean Safety (Remploy brand)",
+        "value": "ocean safety-remploy-brand"
```

Comment:
Looks like there's still a space here when there should be a dash?

Reply:
Make it stop.

Reply:
I did fix a -brand one, but it was ANOTHER ONE.